test/unit/Cover/buyCover.js

```diff
@@ -2,7 +2,6 @@ const { expect } = require('chai');
 const { ethers } = require('hardhat');
```

Comment:
i'm not seeing a test for 

`Should revert if product is not initialized`

should revert with `Cover: Product not initialized`

Reply:
I saw your comment.

The way to test is to refer to some product id that's not used.

Look at how setup adds a product which will have index 0.

https://github.com/NexusMutual/smart-contracts/blob/568e2f6525147789bb54924afa976dc69de899b2/test/unit/Cover/setup.js#L188

Refer to the one with productId: 1 to make it crash

Reply:
https://github.com/NexusMutual/smart-contracts/blob/568e2f6525147789bb54924afa976dc69de899b2/test/unit/Cover/setup.js#L188

To make it revert with that message add a product like in the setup, which will have productId = 1

and set initialPriceRatio: 0

Reply:
Thanks for the help!

I tried doing that but it reverts with `Cover: initialPriceRatio < GLOBAL_MIN_PRICE_RATIO`  which is set as a constant `GLOBAL_MIN_PRICE_RATIO = 100`.

Unless I'm missing something, it looks like products could not be in an uninitialized state, so in that case, Is that validation needed? I think @roxdanila mentioned something about new functionality to deprecate products in which case once introduced it would make sense to validate that the product was not deprecated

Reply:
Ah i see you're right. That validation was added very recently and that require is not useful anymore.

Thanks for clarifying.